{{cookiecutter.repo_name}}/requirements/production.txt

```diff
@@ -1,7 +1,7 @@
 # Packages required in a production environment
 -r base.txt
 
-gevent==1.0.2
-gunicorn==19.3.0
-MySQL-python==1.2.5
-PyYAML==3.11
+gevent
+gunicorn
```

Comment:
At this point the latest version of either library is Python 3.5 compatible. I'd be really surprised if developers chose anything besides a recent version.